assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font-family set in universal selector */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+/* Navbar styling */
+
+.navbar {
+  padding: 20px;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.navbar nav {
+  padding-top: 15px;
+  margin-right: 20px;
+  float: right;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 20px;
+}
+
+.navbar nav ul > li {
+  display: inline-block;
+  margin-left: 25px;
+  list-style-type: none;
+}
+
+.navbar h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.navbar h1 .seo {
+  color: #d9dcd6;
+}
+
```

Comment:
This does work, but I would put the `h1` styling before the `ul`/`li` as the `h1` comes first in the navbar.